enhancements/console/dynamic-plugins.md

```diff
@@ -149,16 +149,27 @@ spec:
 
 In case the plugin needs to communicate with some in-cluster service, it can
 declare a service proxy in its `ConsolePlugin` resource using the
-`spec.proxy.services` array field. A service `name`, `namespace` and `port`
-needs to be specified.
+`spec.proxy.services` map. The key of the map is the proxy alias
+of the service and value is an object in which service `name`, `namespace` and `port`
+needs to be specified and to which the request will be proxied.
+
+```yaml
+spec:
+  proxy:
+    services:
+      <proxy-alias>:
```

Comment:
Nit: this is really an alias for the service rather than for the proxy itself.

Reply:
then I guess it should be `service-proxy-alias` ?